### DISCUSSION TITLE
entire network can not access github

### DISCUSSION THREAD

**Discussion by Alethio36** [community/community#62534] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

this may seem odd, but does github have rate limits or anything like that? I cant access their service for some reason. no error just a connection error page or in command line, timeouts

1) purged dns cache and restarted unbound 
2) took the ip to a different network and it works on the other network
3) grabbed the ip from a different region and added a dns entry into my network and it works on home network now
4) restarted all affected machines (basically all of them in my network)
5) trace rt seems to show traffic leaving my network and isp network (maybe?)
6) i found another issue raised that resolved by adjusting the MTU. in my firewall (pfsense) interfaces i manually adjusted the mtu to 1500 and that did not resolve the issue

im running windows and linux in my environment

**Comment by Alethio36** [comment, marked as answer]
and now its randomly working again. im guessing a isp issue at this point.

**Reply by TomLikeGo** [reply]
didn't git hub block IPs?

**Comment by TomLikeGo** [comment]
i got the same problem for a couple of days and had to forward the traffic to a different IP then it worked. don't why

**Reply by kartikeysaini** [reply]
how did you do that?
